packages/cyberstorm/src/components/Layout/PackageListLayout/PackageListLayout.tsx

```diff
@@ -89,13 +84,17 @@ export function PackageListLayout(props: PackageListLayoutProps) {
               colorScheme="accent"
               size="bold_large"
             />,
-            <Link
-              key="meta-link"
-              leftIcon={<FontAwesomeIcon icon={faDiscord} fixedWidth />}
-              label="Join our community"
-              externalUrl="https://discord.thunderstore.io/"
-              size="bold"
-            />,
+            <a key="meta-link" href="https://discord.thunderstore.io/">
```

Comment:
Buttons aren't supposed to be nested inside anchors. We probably want to keep some sort of `Link` element for links leading outside the website.